Sources/CentralManager/CentralManager.swift

```diff
@@ -241,6 +241,10 @@ extension CentralManager.DelegateWrapper: CBCentralManagerDelegate {
             } catch {
                 Self.logger.error("Received onDidConnect without a continuation!")
```

Comment:
Since now we expect connections that aren't triggered by the central manager's `connect` function, we should demote this `error` to a `warning`.